tests/framework/PostProcessors/SparseSensing/tests

```diff
@@ -7,4 +7,12 @@
     required_libraries = 'imageio'
     rel_err = 0.01
   [../]
+  [./testSPSLOptiTwistClassification]
+    type = 'RavenFramework'
+    input = 'testSPSLOptiTwistClassification.xml'
+    csv = 'classificationOptiTwist/outPP.csv'
+    image = 'classificationOptiTwist/mySensorPlot_scatter-scatter.png classificationOptiTwist/mySensorPlot3D_scatter.png'
+    required_libraries = 'imageio'
+    rel_err = 0.01
+  [../]
```

Comment:
This will only review the image with a relative error of 0.01 which is too small for image comparison. We need to test the csv and the image. `rel_err` for image can be 0.1